test/conformance/serial/ptp.go

```diff
@@ -533,6 +533,43 @@ var _ = Describe("["+strings.ToLower(DesiredMode.String())+"-serial]", Serial, f
 
 			})
 		})
+		Context("Running with reference plugin", func() {
+			BeforeEach(func() {
+				By("Enabling reference plugin", func() {
+					Expect(ptphelper.EnablePTPReferencePlugin()).NotTo(HaveOccurred())
+					ptphelper.WaitForPtpDaemonToBeReady()
+				})
+			})
+			AfterEach(func() {
+				By("Disabling reference plugin", func() {
+					Expect(ptphelper.DisablePTPReferencePlugin()).NotTo(HaveOccurred())
+					ptphelper.WaitForPtpDaemonToBeReady()
+				})
+			})
+			It("Should check whether PTP operator needs to enable reference plugin", func() {
+				By("Find if variable set to enable reference plugin")
+				Expect(ptphelper.EnablePTPReferencePlugin()).NotTo(HaveOccurred())
+			})
+			It("Should check whether plugin is loaded", func() {
+				By("checking for plugin logs")
+				ptpPods, err := client.Client.CoreV1().Pods(pkg.PtpLinuxDaemonNamespace).List(context.Background(), metav1.ListOptions{LabelSelector: "app=linuxptp-daemon"})
+				Expect(err).NotTo(HaveOccurred())
+				Expect(len(ptpPods.Items)).To(BeNumerically(">", 0), "linuxptp-daemon is not deployed on cluster")
+				pluginLog := "Trying to register plugin: generic"
+				foundMatch := false
+				for podIndex := range ptpPods.Items {
+					_, err := pods.GetPodLogsRegex(ptpPods.Items[podIndex].Namespace,
+						ptpPods.Items[podIndex].Name, pkg.PtpContainerName,
+						pluginLog, true, pkg.TimeoutIn3Minutes)
+					if err != nil {
+						logrus.Errorf(fmt.Sprintf("Reference plugin not loaded, err=%s", err))
+						continue
+					}
+					foundMatch = true
```

Comment:
I don't think GetPodLogsRegex returning err==nil means a match is found.
Could you fix the function GetPodLogsRegex()?
https://github.com/openshift/ptp-operator/blob/master/test/pkg/pods/pods.go#L182
This last line should be
```
return matches, fmt.Errorf(""no matches found")
```

Reply:
It will return this error if there's no match:
https://github.com/openshift/ptp-operator/blob/master/test/pkg/pods/pods.go#L161

Don't want to make any changes there as part of this PR so I don't break anything else; it can be done in a followup PR if necessary